templates/manage_certs.py

```diff
@@ -32,7 +33,7 @@ def get_all_domains(config):
             line = line.strip()
             if line and not line.startswith("#"):
                 domain, backend = line.split(None, 1)
-                domains[domain] = backend
+                domains[domain.lower()] = backend
```

Comment:
@smarnach Thank you for the unit tests. That's true that for some of the functions, relying on the integration tests is enough, and will provide better guarantees than a unit test. Having both though helps to add granularity to the blanket testing from the integration tests, and having the infrastructure in place will helps others working on this in the future - when there isn't any, it becomes very tempting to not write unit tests.